src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMParams.scala

```diff
@@ -17,6 +17,15 @@ trait LightGBMExecutionParams extends Wrappable {
   def getParallelism: String = $(parallelism)
   def setParallelism(value: String): this.type = set(parallelism, value)
 
+  val topK = new Param[Int](this, "topK",
+    "The top_k value used in Voting parallel, " +
+      "set this to larger value for more accurate result, but it will slow down the training speed. " +
+      "It should be greater than 0")
+  setDefault(topK -> LightGBMConstants.DefaultTopK)
```

Comment:
interesting idea to put these in constants, I like it

Reply:
Thanks for your review